CromIAM/src/main/resources/swagger/cromiam.yaml

```diff
@@ -424,43 +424,45 @@ paths:
           description: Internal Error
       security:
         - googleoauth:
-            - openid
-            - email
-            - profile
+          - openid
+          - email
+          - profile
   '/api/workflows/{version}/{id}/releaseHold':
-        post:
-          summary: Switch a workflow from 'On Hold' to 'Submitted' status
-          description: Request Cromwell to release the hold on a workflow. It will switch the status of a workflow from 'On Hold' to 'Submitted' so it can be picked for running. For instance this might be necessary in cases where you have submitted a workflow with workflowOnHold = true.
-          parameters:
-            - name: version
-              description: API Version
-              required: true
-              type: string
-              in: path
-              default: v1
-            - name: id
-              description: A workflow ID
-              required: true
-              type: string
-              in: path
-          tags:
-            - Workflows
-          responses:
-            '200':
-              description: Successful Request
-              schema:
-                $ref: '#/definitions/WorkflowStatusResponse'
-            '400':
-              description: Malformed Workflow ID
-            '403':
-              description: Malformed Request
-            '404':
-              description: Workflow ID Not Found
-            '500':
-              description: Internal Error
-          security:
-            - google_oauth:
-                - openid
+    post:
+      summary: Switch a workflow from 'On Hold' to 'Submitted' status
+      description: Request Cromwell to release the hold on a workflow. It will switch the status of a workflow from 'On Hold' to 'Submitted' so it can be picked for running. For instance this might be necessary in cases where you have submitted a workflow with workflowOnHold = true.
+      parameters:
+        - name: version
+          description: API Version
+          required: true
+          type: string
+          in: path
+          default: v1
+        - name: id
+          description: A workflow ID
+          required: true
+          type: string
+          in: path
+      tags:
+        - Workflows
+      responses:
+        '200':
+          description: Successful Request
+          schema:
+            $ref: '#/definitions/WorkflowStatusResponse'
+        '400':
+          description: Malformed Workflow ID
+        '403':
+          description: Malformed Request
+        '404':
+          description: Workflow ID Not Found
+        '500':
+          description: Internal Error
+      security:
+        - googleoauth:
+          - openid
+          - email
+          - profile
```

Comment:
For anyone checking this is the real change beside fixing the indentation.

"Diff settings" > "Hide whitespace changes" also helps too.